sections/appendix/problems.tex

```diff
@@ -0,0 +1,30 @@
+\subsection{Encountered Problems}
+
+\subsubsection{Too many low values}
+The document-topic distributions seem to have too low entropy and a lot of very low values.
+This causes each topic to have an average of $\sim$7000 docs in their distribution, however with very low entropy, meaning that some of these distributions are very insignificant, but add space and calculation time to our sparse representations.
+To fix this we try tuning a minimum threshold for a topic to be counted in the document-topic distributions, which can be set as a hyper-parameter for \emph{gensim.ldamodel.get\_document\_topics()}.
+After setting up a minimum percent of 2.5\% for a entry to be included in the document distribution, the amount of documents per topic fell to ~4800, removing more than 5 entries per document for a total of $\sim$350.000 entries deleted. This has overall made the matrices a lot more sparse, but we will still have to evaluate whether changing K is better.
+
+\subsubsection{Empty Topics}
+There are also a significant number of topics, which had words, but no assigned docs. These should be detected and cut away, though their existence points towards a lower $K$ might be needed.
+
+\subsubsection{Too Common Topics}
+On the opposite end we also have some topics which are so universal that almost all documents are included in this topic. This might be a sign that pre-processing is not working well enough, or it might once again point at the needed for a better K. Alternatively, too general topics could be manually pruned after the LDA model has been run.
```

Comment:
```suggestion
On the opposite end, we also have some topics which are so universal that almost all documents are included in this topic. 
This might be a sign that pre-processing is not working well enough, or it might once again point at the needed for a better K.
Alternatively, too general topics could be manually pruned after the LDA model has been run.
```